che-core-vfs-impl/src/main/java/org/eclipse/che/vfs/impl/fs/FSMountPoint.java

```diff
@@ -297,6 +249,11 @@ protected AccessControlList loadValue(Path key) {
     private final VirtualFileSystemUserContext userContext;
 
     /**
+     * ACL that returned for any fs entry.
+     */
+    private final AccessControlList defaultAcl = new AccessControlList();
```

Comment:
As we expect in near future to remove this code at all I would say such complexity is not necessary. The purpose of this change is to make things working until we remove VFS at all.